src/connectionController.ts

```diff
@@ -248,10 +248,12 @@ export default class ConnectionController {
 
   public sendTelemetry(
     newDataService: DataServiceType,
+    connectionModel: ConnectionModel,
     connectionType: ConnectionTypes
   ): void {
     this._telemetryService.trackNewConnection(
-      newDataService,
+      newDataService.client.client,
```

Comment:
Maybe we can think in a feature about a better naming instead `client.client`. I saw it before and it every time confuses me :)